specification/async_search/submit/AsyncSearchSubmitRequest.ts

```diff
@@ -151,6 +152,11 @@ export interface Request extends RequestBase {
      * names matching these patterns in the hits.fields property of the response.
      */
     docvalue_fields?: FieldAndFormat[]
+    /**
+     * Defines the approximate kNN search to run.
+     * @since 8.4.0
+     */
+    knn?: KnnQuery
```

Comment:
Should it be added to `MultisearchBody`?

Reply:
Yes! I've done so, nice catch.